packages/amplify_core/test/config/unknown_test.dart

```diff
@@ -35,7 +36,7 @@ void main() {
       };
 
       final config = AmplifyConfig.fromJson(json);
-      const expected = AmplifyConfig(
+      var expected = AmplifyConfig(
```

Comment:
*nit:* If this cannot be `const` I would make it `final`
```suggestion
      final expected = AmplifyConfig(
```